continuous_integration/hdfs/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:trusty
+
+RUN apt-get update && \
+    apt-get install -y -q curl bzip2 git && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install CDH5 in a single node: Pseudo Distributed
+# Docs: http://www.cloudera.com/content/www/en-us/documentation/enterprise/latest/topics/cdh_qs_yarn_pseudo.html
+ADD docker-files/cloudera.pref /etc/apt/preferences.d/cloudera.pref
+RUN curl -s http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh/archive.key | apt-key add - && \
+    echo 'deb [arch=amd64] http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh5 contrib' > /etc/apt/sources.list.d/cloudera.list && \
+    echo 'deb-src http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh5 contrib' >> /etc/apt/sources.list.d/cloudera.list && \
+    apt-get update && \
+    apt-get install -y -q openjdk-7-jre-headless hadoop-conf-pseudo && \
```

Comment:
What is hadoop-conf-pseudo ?
FYI: conda's java is probably sufficient (but no need to change)

Reply:
This whole install process is from before, and I believe copied mostly verbatim from the cloudera docs here: https://www.cloudera.com/documentation/enterprise/5-5-x/topics/cdh_qs_yarn_pseudo.html. `hadoop-conf-pseudo` is the pseudo-distributed configuration.